MAUI/PDF-Viewer/Form-Filling.md

```diff
@@ -251,6 +251,279 @@ private void PdfViewer_FormFieldValueChanged(object sender, FormFieldValueChange
+
+## Hide form fields
+
+To hide form fields in the document, set the `IsHidden` property of the form fields to `true`.
+
+### Hide all form fields
+
+While reviewing and presenting documents, all form fields can be hidden to improve the focus on the content and to avoid distraction caused by cluttered form fields. The following example explains how to hide all form fields in a PDF document. 
+
+{% tabs %}
+{% highlight xaml %}
+<Grid> 
+    <Grid.RowDefinitions> 
+        <RowDefinition Height="Auto" /> 
+        <RowDefinition Height="*" /> 
+    </Grid.RowDefinitions> 
+    <Button Text="HideAllFormFields" Clicked="HideAllFormFieldsButton_Clicked" VerticalOptions="Center" HorizontalOptions="Center" Grid.Row="0"/> 
+    <syncfusion:SfPdfViewer x:Name="PdfViewer" DocumentSource="{Binding PdfDocumentStream}" Grid.Row="1" /> 
+</Grid> 
+{% endhighlight %}
+
+{% highlight C# %}
+private void HideAllFormFieldsButton_Clicked(object sender, EventArgs e) 
+{ 
+    foreach(FormFields  formfield in PdfViewer.FormFields) 
+    { 
+       formfield.IsHidden = true; 
+    } 
+} 
+{% endhighlight %}
+{% endtabs %}
+
+* Similarly, to show all the form fields, set the `IsHidden` property value to `false`.
+
+### Hide specific form fields 
+
+While collaborating, the `IsHidden` property of form fields can be used to show and hide form fields from specific authors. The following example illustrates how to hide form fields from a specific author by accessing the form field’s`IsHidden` property. 
+
+{% tabs %}
+{% highlight xaml %}
+<Grid> 
+    <Grid.RowDefinitions> 
+        <RowDefinition Height="Auto" /> 
+        <RowDefinition Height="*" /> 
+    </Grid.RowDefinitions> 
+    <Button Text="HideFormField" Clicked="HideFormFieldButton_Clicked" VerticalOptions="Center" HorizontalOptions="Center" Grid.Row="0"/> 
+    <syncfusion:SfPdfViewer x:Name="PdfViewer" DocumentSource="{Binding PdfDocumentStream}" Grid.Row="1" /> 
+</Grid> 
+{% endhighlight %}
+
+{% highlight C# %}
+private void HideFormFieldButton_Clicked(object sender, EventArgs e) 
+{ 
+FormField formField = pdfViewer.FormFields.Where(x => x.Name == "name").FirstOrDefault();
+
+if (formField is TextFormField nameTextBox)
+{
+    // Hide  the Text Box field.
+    nameTextBox.IsHidden = true;
+}
+}
+{% endhighlight %}
+{% endtabs %}
+
+* Similarly, to show the form fields, set the `IsHidden` property value to `false`. 
+
+# Form field associated with widget
```

Comment:
Headings are not correct. Can you please confirm this content is created for which feature?
<img width="500" alt="image" src="https://github.com/user-attachments/assets/a6910d63-9310-4ce4-b7ac-e8a36eddf864" />